RFCs/0023-TestSettings-Deprecation.md

```diff
@@ -0,0 +1,31 @@
+# 0023 TestSettings Deprecation
+
+## Summary
+
+This note outlines the proposed changes to deprecate support for testsettings in test runs.
+
+## Motivation
+
+Today there are two types of files to configure tests: *.testsettings and *.rusnettings. To simplify the experience, we are planning to deprecate support for *.testsettings.
+
+## RoadMap
+
```

Comment:
This might increase the scope of this RFC.

LegacySettings is not something we want to encourage as such. Ideally, we want users to move away from testsettings and use first class supported runsettings node.
Legacy should be used only if they are not able to achieve that using the existing settings.

For example, for parallel, user should actually try MaxCPUCount or/and MSTest parallelize instead of simply using legacy.